<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Utils::WizardPage</class>
 <widget class="QWizardPage" name="Utils::WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>68</height>
   </rect>
  </property>
  <property name="title">
   <string>Choose the Location</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::FileNameValidatingLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::PathChooser" name="pathChooser" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FileNameValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/filenamevalidatinglineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
